includes/rest/class-following.php

```diff
@@ -72,7 +72,7 @@ public static function get( $request ) {
 		$json->actor = \get_author_posts_url( $user_id );
 		$json->type = 'OrderedCollectionPage';
 
-		$json->partOf = \get_rest_url( null, "/activitypub/1.0/users/$user_id/following" ); // phpcs:ignore
+		$json->partOf = get_rest_url_by_path( sprintf( 'users/%d/following', $user_id ) ); // phpcs:ignore
```

Comment:
`get_rest_url_by_path` is in the `Activitypub` namespace, so you have to add `use function Activitypub\get_rest_url_by_path` as the top of the class.